plugins/modules/azure_rm_manageddisk.py

```diff
@@ -172,6 +172,11 @@
             - The total throughput (MBps) that will be allowed across all VMs mounting the shared disk as ReadOnly.
             - MBps means millions of bytes per second - MB here uses the ISO notation, of powers of 10.
         type: int
+    tier:
+        description:
+            - Performance tier assigned to this disk.
+            - See U(https://learn.microsoft.com/en-us/azure/virtual-machines/disks-change-performance) for more information about disk performance tiers.
+        type: str
```

Comment:
Missing choices;  Argument 'tier' in argument_spec defines choices as (['', 'P1', 'P2', 'P3', 'P4', 'P6', 'P10', 'P15', 'P20', 'P30', 'P40', 'P50', 'P60', 'P70', 'P80']) 

        
        
        
```

Reply:
I have added improved description and choices.